LICENSE

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2017, conda-forge
-All rights reserved.
```

Comment:
I understand now what you were referring to. Yes, I was moving files around and should not have deleted the `LICENSE` file from the `staged-recipes` directory. Are you planning to just close this PR and let [#2914](https://github.com/conda-forge/staged-recipes/pull/2914) replace?

Thanks.